### DISCUSSION TITLE
0.16.0

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#336] [Announcements]
详细更新日志请参考 [CHANGELOG.md](https://github.com/FightingDesign/fighting-design/blob/master/CHANGELOG.md)。

Please refer to [CHANGELOG.en-US.md](https://github.com/FightingDesign/fighting-design/blob/master/CHANGELOG.en-US.md) for details.

<hr /><em>This discussion was created from the release <a href='https://github.com/FightingDesign/fighting-design/releases/tag/0.16.0'>0.16.0</a>.</em>

**Comment by keer-tea** [comment]
田同学，每次提交信息的三个表情，你有什么方法快速生成吗？还是去表情网站随便复制粘贴表情，好奇哈哈

**Reply by Tyh2001** [reply]
目前是通过我自己写的 [ markdown emoji](https://tianyuhao.cn/emoji/) 来复制贴到 commit 的 message 中。不过最近我也在研究自动化，每次 pnpm commit 的时候期望可以出发钩子自动添加，理论上是可行的

@long-life233
